rocAL/source/pipeline/master_graph.cpp

```diff
@@ -35,6 +35,7 @@ THE SOFTWARE.
 #include "meta_data_graph_factory.h"
 #include "randombboxcrop_meta_data_reader_factory.h"
 #include "node_copy.h"
+#include "seed_rng.h"
```

Comment:
I think #include "seed_rng.h" is already added in master_graph.h
Need to recinclude here